srcs/http/request/parse/http_parse.cpp

```diff
@@ -32,48 +36,65 @@ bool IsStringUpper(const std::string &str) {
 	return true;
 }
 
+// OWS (optional whitespace): SP(Space), HTAB(Horizontal Tab)
+void TrimLeadingOptionalWhitespace(std::string &str) {
+	typedef std::string::iterator It;
+	for (It it = str.begin(); it != str.end(); ++it) {
+		if (!IsOptionalWhitespace(*it)) {
+			str.erase(str.begin(), it);
+			return;
+		}
+	}
+	str.erase(str.end());
+}
+
 } // namespace
 
 // todo: tmp request_
 HttpRequestResult HttpParse::Run(const std::string &read_buf) {
 	HttpRequestResult result;
 	// a: [request_line header_fields, messagebody]
```

Comment:
```suggestion
	// a: [request_line + header_fields, messagebody]
```
細かいんですけどこっちの方がわかりやすいかもです